src/main/scala/com/exasol/cloudetl/kafka/KafkaConsumerProperties.scala

```diff
@@ -299,6 +299,28 @@ class KafkaConsumerProperties(private val properties: Map[String, String]) exten
     }
   }
 
+  /**
+   * Gets custom krb5.conf file location. Checks that user provided path exists.
+   * @return file path or empty string
+   */
+  final def getSASLKrb5ConfLocation(): String = {
+    val krb5confLocation = get(SASL_KRB5CONF_LOCATION.userPropertyName)
+    if (krb5confLocation.isEmpty) {
+      ""
+    } else {
+      if (!Files.isRegularFile(Paths.get(krb5confLocation.get))) {
+        throw new KafkaConnectorException(
+          ExaError
+            .messageBuilder("E-KCE-28")
+            .message("Unable to find the custom krb5.conf file at {{JAAS_LOCATION}}", krb5confLocation)
+            .mitigation(BUCKETFS_CHECK_MITIGATION)
```

Comment:
Beware: Java compiler will not allow using constants in mitigations etc. 
But for scala (luckily or unluckily ?) there is no compiler :slightly_smiling_face: